### ISSUE TITLE
Play !byefelicia whenever a user leaves the stream

### ISSUE THREAD

**Issue by ToeFrog** [clarkio/ttv-chat-light#105]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)

- [ ] bug report -> please search issues before submitting
- [ x] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)

### Minimal steps to reproduce
>

### Any log messages given by the failure
>

### Expected/desired behavior
> Play 'byefelicia' when a user leaves the stream and possibly have the ability for custom sounds for specific users similar to when users join.

### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)

### Versions
>

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by ToeFrog** [comment]
@clarkio Is the ttvPart function on line 116 in src > server > twitch-chat.ts the function that would need a change here? Looks like we would need something in the effects-manager.ts file that could be called like this.effectsManager.activatePartEffect() to play for whenever someone leaves. I'm assuming Part in this case is the same as someone leaving the stream.

**Comment by clarkio** [comment]
Yes that is the correct event listener function we'd want to use for this.

**Comment by ToeFrog** [comment]
PR submitted

**Comment by clarkio** [comment]
Referencing PR #119 to keep track of the thoughts and ideas on how to approach this in a way that doesn't leave people feeling bad for leaving the stream. Going to close that PR for now and leave this issue open.

**Comment by ToeFrog** [comment]
I agree with not merging this. I could see how it could be seen poorly or distracting for the streamer as well as those watching the stream.
